packages/react/src/index.js

```diff
@@ -117,6 +117,7 @@ export DynamicHotspotSourcePicker from './components/HotspotEditorModal/DynamicH
 export ComboChartCard from './components/ComboChartCard/ComboChartCard';
 export MenuButton from './components/MenuButton/MenuButton';
 export ListBuilder from './components/ListBuilder/ListBuilder';
+export TearSheetWrapper from './components/TearSheet/TearSheet';
```

Comment:
This is actually exporting the `TearSheet`, not the `TearSheetWrapper`, and we'll need to export both components, right?

Reply:
Right! Done 👍